### PR TITLE
ITC-201: added system status aka lumturio

### PR DESCRIPTION
**Pull request by lilosti**

### CODE REVIEW

File: config/sync/views.view.upcoming_events.yml [hunk 1]
```diff
@@ -634,11 +634,11 @@ display:
             default_group_multiple: {  }
             group_items:
               1:
-                title: 'Igangværende og kommende events'
+                title: 'Ongoing and upcoming events'
```
Comment (turegjorup) [inline config/sync/views.view.upcoming_events.yml]: Is the switch from `da` to `en` intentional?

File: config/sync/views.view.upcoming_events.yml [hunk 2]
```diff
@@ -634,11 +634,11 @@ display:
...
                 operator: in
                 value: '1'
               2:
-                title: 'Afsluttede events'
+                title: 'Past events'
```
Comment (turegjorup) [inline config/sync/views.view.upcoming_events.yml]: Is the switch from `da` to `en` intentional?

File: config/sync/system_status.settings.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+system_status_token: 55348469994162l
+system_status_encrypt_token: D79hL939883
```
Comment (turegjorup) [inline config/sync/system_status.settings.yml]: `encrypt_token` sounds like something that should not be in git. Can a 3rd party impersonate calls on our behalf with these tokens?

Comment (turegjorup) [reply config/sync/system_status.settings.yml]: If we deploy this to both `stg` and `prod` can the lumtrio service distinguish between the two?

**Review by turegjorup — COMMENTED** [review]
What is the scope for this PR? Branch name is "Go Live", PR name is "system status". If scope is "system status" then a lot of the config files are out of scope, and should not be included.

PR is marked as "ready for review" but there is still commits being pushed to it. Please convert to draft status until done.

File: config/sync/system_status.settings.yml (hunk 3 above)
Comment (lilosti) [reply config/sync/system_status.settings.yml]: The UUID is a combination of these tokens, I have no idea on how this works

Comment (turegjorup) [reply config/sync/system_status.settings.yml]: If you search for `system_status_token` under the `aakb` org on github you can see how we do on other sites like: https://github.com/aakb/cfia/blob/d47f841dc11ec87514f26dcda974b957d32dfbc9/config/sync/system_status.settings.yml

Comment (turegjorup) [reply config/sync/system_status.settings.yml]: This needs to be moved to `settings.local.php`

**Review by turegjorup — CHANGES_REQUESTED** [review]

**Review by turegjorup — APPROVED** [review]
Approved. Please note that this can not be fully deployed automatically, `settings.local.php` needs to be updated manually.

**Pull request merged**